Makefile.kokkos

```diff
@@ -736,6 +737,13 @@ ifeq ($(KOKKOS_INTERNAL_USE_ARCH_ARMV81), 1)
   endif
 endif
 
+ifeq ($(KOKKOS_INTERNAL_USE_ARCH_A64FX), 1)
+  tmp := $(call kokkos_append_header,"$H""define KOKKOS_ARCH_A64FX")
+
+  KOKKOS_CXXFLAGS += -march=armv8.2-a+sve
+  KOKKOS_LDFLAGS += -march=armv8.2-a+sve
+endif
+
```

Comment:
Do we really need makefile support for _new_ architectures? This is definitely extra maintenance work and it's not really tested in our CI AFAICT.

Reply:
Yes we do.